rocAL/rocAL/include/loaders/image/image_read_and_decode.h

```diff
@@ -81,7 +81,7 @@ class ImageReadAndDecode
     std::shared_ptr<Reader> _reader;
     std::vector<std::vector<unsigned char>> _compressed_buff;
     std::vector<size_t> _actual_read_size;
-    std::vector<std::string> _image_names;
+    std::vector<std::string> _sample_names;
```

Comment:
Why is this changed?
It is specific to image_read_and_decode.cpp so we can have image names itself

Reply:
Reverted this change.